### ISSUE TITLE
Docking clearance is granted for a different pad # than the actual pad # outlined for landing

### ISSUE THREAD

**Issue by clausimu** [pioneerspacesim/pioneer#4016]
### Observed behaviour
I was granted docking clearance at Itzalean (New Hope) for pad no. 9 (two other ships were docked), but the actual pad that I needed to land at was no. 10 (as outlined by the target marker). I have observed this several times now, so I guess it must be an actual bug.

### Expected behaviour
Grant docking clearance for the pad no. that is actually the targeted landing site.

### Steps to reproduce
Try to dock with Itzalean (when other ships are present) and compare the pad no. that you were cleared to land at with the one that is actually the target.

__My pioneer version (and OS):__
pioneer version: git build 7c64d721c
OS: openSUSE 64bit 42.1

**Comment by fluffyfreak** [comment]
Sounds like an off-by-one issue, I wonder if pads are numbered 1-10 but the code is 0-9

**Comment by ecraven** [comment]
> Sounds like an off-by-one issue, I wonder if pads are numbered 1-10
> but the code is 0-9

fwiw, I've done a bit of manual flying in the Sol system, and never
noticed this when docking manually :-/
I'll try again in the next few days.

**Comment by kennworl** [comment]
I'm on build 1.85 (built from source).  I started a new game at Itzalean with two other ships already docked on pad 7 and pad 2.   I lifted off and hovered, requested docking and got told pad 8.  That pad is blinking orange like you would expect, the occupied pads are blinking white.  When I landed, on pad 8, the lights turned red, and then green again when I lifted off.  Same deal at Los Angeles.  Is there particular pattern or reliable way of replicating this problem?
